### PR TITLE
Fix article in tooltip on update page.

### PR DESCRIPTION
**Pull request by relrod**
Signed-off-by: Rick Elrod <rick@elrod.me>

### CODE REVIEW

File: bodhi/server/util.py [hunk 1]
```diff
@@ -713,8 +718,8 @@ def type2icon(context, kind):
     }.get(kind)
 
     return "<span class='label label-%s' data-toggle='tooltip'\
-            title='This is a %s update'><i class='fa fa-fw %s'></i></span> \
-            " % (cls, kind, fontawesome)
+            title='This is %s %s update'><i class='fa fa-fw %s'></i></span> \
+            " % (cls, kind_article, kind, fontawesome)
```
Comment (bowlofeggs) [inline bodhi/server/util.py]: Looks like we just need to add some test coverage to this to get CI to be happy. You can add the tests at ```bodhi.tests.server.test_util```. I think we could probably get by with two tests. Here's some untested handy-wavy pseudocode to get you started:

```
def test_consonant(self):
    """Test type2icon() with a kind that starts with a consonant."""
    self.assertEqual(util.type2icon('security'), 
                               ("<span class='label label-%s' data-toggle='tooltip'"
                                "title='This is a security update'><i class='fa fa-fw danger'></i></span> "))

def test_vowel(self):
    """Test type2icon() with a kind that starts with a vowel."""
    self.assertEqual(util.type2icon('enhancement'), 
                               ("<span class='label label-%s' data-toggle='tooltip'"
                                "title='This is an enhancement update'><i class='fa fa-fw success'></i></span> "))
```

**Review by bowlofeggs — CHANGES_REQUESTED** [review]

**Comment by bowlofeggs** [comment]
Hi @relrod, do you still want to work on this?

**Comment by bowlofeggs** [comment]
I am going to add these two tests.

**Review by bowlofeggs — APPROVED** [review]

**Comment by relrod** [comment]
+1

**Comment by bowlofeggs** [comment]
jenkies test

**Comment by bowlofeggs** [comment]
jenkies test

**Comment by bowlofeggs** [comment]
These patches are planned to be included in the upcoming 3.10.0 release: #2556.

**Comment by bowlofeggs** [comment]
These patches have been deployed to Fedora's staging Bodhi instance:

https://bodhi.stg.fedoraproject.org/

**Pull request merged**